macroid-core/src/main/scala/macroid/Contexts.scala

```diff
@@ -45,12 +46,19 @@ object ContextWrapper {
 
   def apply[F](fragment: F)(implicit fragmentImpl: Fragment[F]): ActivityContextWrapper =
     ContextWrapper(fragmentImpl.activity(fragment))
+
+  def apply(view: View): ContextWrapper = view.getContext match {
+    case activity: Activity => ContextWrapper(activity)
+    case service: Service => ContextWrapper(service)
+    case app: Application => ContextWrapper(app)

```

Comment:
I think this makes sense, but I’d call it `GenericContextWrapper` instead. And the above match could be left in place, it just needs a default case at the end with `GenericContextWrapper`.